### PR TITLE
Use proper comment syntax in servo.css

### LINKED ISSUE #11963: servo.css contains incorrect comment syntax
https://github.com/servo/servo/blob/master/resources/servo.css#L135 uses `//` for commenting rather than `/*...*/`.

### PR DESCRIPTION
**Pull request by cbrewster**
---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix #11963 (github issue number if applicable).

- [ ] There are tests for these changes OR
- [X] These changes do not require tests because using proper comment syntax in css.

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/11964)

### CODE REVIEW

**Comment by asajeffrey** [comment]
This can be r=me, but lets wait until tomorrow to do it.

File: resources/servo.css [hunk 1]
```diff
@@ -132,7 +132,7 @@ textarea {
   -servo-overflow-clip-box: content-box;
 }
 
-// https://html.spec.whatwg.org/multipage/rendering.html#the-details-and-summary-elements
```
Comment (notriddle) [inline resources/servo.css]: ... I'm so sorry for writing that.

Comment (asajeffrey) [inline resources/servo.css]: No problem, if I had to remember the comment syntax in every language I use, I don't think my brain would have space for much else!

**Comment by ConnorGBrewster** [comment]
@bors-servo retry
 * yolo fallout

**Pull request merged**